src/java/main/org/apache/zookeeper/client/ConnectStringParser.java

```diff
@@ -20,8 +20,11 @@
 
 import java.net.InetSocketAddress;
 import java.util.ArrayList;
+import java.util.List;
 
 import org.apache.zookeeper.common.PathUtils;
+import static org.apache.zookeeper.common.StringUtils.split;
```

Comment:
Static import of individual member is allowed as it sometimes improves the readability
Please let me know your opinion on this and if required I will change it.